### DISCUSSION TITLE
nav_mc_hover_thr & nav_use_midthr_for_althold

### DISCUSSION THREAD

**Discussion by nullx8** [iNavFlight/inav#7383] [Q&A]
is nav_mc_hover_thr still relevant when nav_use_midthr_for_althold is ON ? 

and to clarify nav_use_midthr_for_althold = ON means it uses thr_mid as nav_mc_hover_thr ) ?

**Comment by avsaase** [comment]
> is nav_mc_hover_thr still relevant when nav_use_midthr_for_althold is ON ?

Yes

> and to clarify nav_use_midthr_for_althold = ON means it uses thr_mid as nav_mc_hover_thr ) ?

No. 

'nav_use_midthr_for_althold = ON' means you use the throttle stick center position as the "keep altitude" command. When set to 'OFF' you use the throttle stick position from the moment you enabled altitude hold.

**Reply by nullx8** [reply]
> > is nav_mc_hover_thr still relevant when nav_use_midthr_for_althold is ON ?
> Yes

How ?

> > and to clarify nav_use_midthr_for_althold = ON means it uses thr_mid as nav_mc_hover_thr ) ?
> No.
> 'nav_use_midthr_for_althold = ON' means you use the throttle stick center position as the "keep altitude" command. When set to 'OFF' you use the throttle stick position from the moment you enabled altitude hold.

lookupThrottleRCMid is used to set the value in setupMulticopterAltitudeController () 
in main/navigation/navigation_multicopter.c#L167

as 
lookupThrottleRCMid = minThrottle + (int32_t)(motorConfig()->maxthrottle - minThrottle) * controlRateConfig->throttle.rcMid8 / 100; 

(throttle.rcMid8 is the container for thr_mid) 
so only if thr_mid is set to 50 the center stick would be the case, also this setting would be completely pointless
as you could just set  nav_mc_hover_thr to 1500 to archive that.
